RELEASE-NOTES.md

```diff
@@ -1,5 +1,20 @@
 # Prefect Release Notes
 
+## Release 2.13.4
+### Enhancements
+
+- Allow task runs without a flow run — https://github.com/PrefectHQ/prefect/pull/10816
```

Comment:
I think something that emphasizes that it's not really user-friendly yet, such as the phrase "laying groundwork" or equivalent, might help

Reply:
awesome, thanks!